source/base/TimeSpan.ooc

```diff
@@ -21,6 +21,7 @@ TimeSpan: cover {
 	toHours: func -> Long { this ticks / DateTime ticksPerHour }
 	toDays: func -> Long { this ticks / DateTime ticksPerDay }
 	toWeeks: func -> Long { this ticks / DateTime ticksPerWeek }
+	inRange: func (start, end: This) -> Bool { this >= start && this <= end }
```

Comment:
This is fine. I will use it for sanity checking metadata, and almost all of them are `TimeSpan`